src/interfaces/token.ts

```diff
@@ -113,6 +113,35 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
       try {
         const zapperBalances = await this.yearn.services.zapper.balances(account);
         balances.zapper = zapperBalances.filter(({ address }) => addresses.zapper.has(address));
+        const balance = await this.ctx.provider.read.getBalance(account);
+        const priceUsdc = await this.yearn.services.oracle.getPriceUsdc(EthAddress);
+        balances.sdk = [
+          {
+            address: EthAddress,
+            token: ETH_TOKEN,
+            balance: balance.toString(),
+            balanceUsdc: new BigNumber(balance.toString())
+              .div(10 ** 18)
+              .times(new BigNumber(priceUsdc))
+              .toString(),
+            priceUsdc,
+          },
+        ];
+
+        const hasWeth = zapperBalances.find(({ address }) => WethAddress === address);
+
+        if (!hasWeth) {
+          balances.sdk = [
+            ...balances.sdk,
+            {
+              address: WethAddress,
+              token: WETH_TOKEN,
+              balance: "0",
+              balanceUsdc: "0",
+              priceUsdc,
+            },
+          ];
+        }
```

Comment:
If WETH is not needed to solve the UI issue, we should just add ETH

Reply:
solved on fd85573125e7eda48038fbb369ac4c7b2f93d0e0